internal/wasm/jit/engine_test.go

```diff
@@ -298,11 +336,6 @@ func TestRelease(t *testing.T) {
 				require.NotContains(t, e.compiledFunctions, f, i)
 			}
 
-			// Release the reference to ModuleEngine -> *compiledFunctions for non-imported compiledFunctions
-			// must be munmapped.
-			modEngine = nil
-			runtime.GC()
```

Comment:
maybe this could be the cause of flake on win

Reply:
ok I'll put it back! I was assuming GC was because we were testing finalization side-effects. guess not heh.

Reply:
actually there's another close later that was still using finalizer..

Reply:
😄